_data/toc.yaml

```diff
@@ -1572,6 +1572,8 @@ manuals:
               title: Use domain names to access services
             - path: /datacenter/ucp/2.1/guides/admin/configure/run-only-the-images-you-trust/
               title: Run only the images you trust
+            - path: /datacenter/ucp/2.1/guides/admin/configure/integrate-with-dtr/
```

Comment:
Can you move this to the 'Configure' section (around line 1551). It makes more sense from an information architecture standpoint.